src/test/java/catan/controllers/game/CreateGameTest.java

```diff
@@ -35,26 +37,22 @@ public void setup() {
     @Test
     public void should_successfully_create_new_private_game() {
         String userToken = loginUser(USER_NAME_1, USER_PASSWORD_1);
-        int userId = getUserId(userToken);
 
         createNewGame(userToken, true, 99)
                 .then()
                 .statusCode(200)
                 .contentType(ACCEPT_CONTENT_TYPE)
-                .body("gameId", greaterThan(0))
-                .body("creatorId", equalTo(userId))
-                .body("privateGame", equalTo(true))
-                .body("status", equalTo(GameStatus.NEW.toString()))
-                .body("privateCode", not(equalTo(isEmptyString())))
-                .body("dateCreated", is(both(   //closeTo can be applied only to 'double' values, but dateCreated is 'long' value
-                        greaterThan(System.currentTimeMillis() - 60000))
-                        .and(
-                                lessThanOrEqualTo(System.currentTimeMillis()))))
-                .body("targetVictoryPoints", equalTo(99))
-                .body("minPlayers", equalTo(TEMPORARY_MIN_PLAYERS))
-                .body("maxPlayers", equalTo(TEMPORARY_MAX_PLAYERS))
-                .body("dateStarted", equalTo(0))
-                .body("gameUsers", not(equalTo(isEmptyString()))); // wtf?
+                .body("gameId", greaterThan(0));
+    }
+
+    @Test
+    public void should_guest_successfully_create_new_private_game() {
+        String userToken = registerAndLoginGuest(USER_NAME_GUEST_1)
+                .path("token");
+
+        createNewGame(userToken, true)
+                .then()
+                .statusCode(200);
```

Comment:
actually you are right, we are asserting that in other test cases, but more assertions is better) we can leave it as is, this is just my purpose